apps/frontend/src/components/common/proposalFilters/InstrumentFilter.tsx

```diff
@@ -17,11 +18,17 @@ const useStyles = makeStyles(() => ({
   },
 }));
 
+export enum InstrumentFilterEnum {
```

Comment:
Is it possible to get it from generated SDK from backend, instead of redefining again in the frontend?

Reply:
I tried but without success because it is not used directly in the graphql schema and it is not exposed while generating. If you have some better suggestion please let me know

Reply:
Btw, is it ok to send MULTI, ALL in the instrument_id parameter itself? Or do you think we should bring in a new parameter to avoid confusion.

Means, instrument_id will always hold the instrument ID, proposal_with_multipe_instrument = TRUE | FALSE, proposal_with_all_instrument = TRUE | FALSE.  Something like that?

This will
1. Not mix up several unrelated data types for the single parameter
2. Easy to understand the code on both frontend and backend.
3. Also solve the ENUM issue we are facing 😁